tsdoc/src/parser/TSDocMessageId.ts

```diff
@@ -0,0 +1,328 @@
+
+  /**
+   * The TSDoc tag ___ is not supported by this tool.
+   */
+  UnsupportedTag = 'tsdoc-unsupported-tag',
+
+  /**
+   * The TSDoc tag ___ is not defined in this configuration.
+   */
+  UndefinedTag = 'tsdoc-undefined-tag',
+
+  /**
+   * The `@param` block should be followed by a parameter name.
+   */
+  ParamTagWithInvalidName = 'tsdoc-param-tag-with-invalid-name',
+
+  /**
+   * The `@param` block should be followed by a parameter name and then a hyphen.
+   */
+  ParamTagMissingHyphen = 'tsdoc-param-tag-missing-hyphen',
+
+  /**
+   * A backslash must precede another character that is being escaped.  OR
+   * A backslash can only be used to escape a punctuation character.
+   */
+  UnnecessaryBackslash = 'tsdoc-unnecessary-backslash',
+
+  /**
+   * Expecting a TSDoc tag starting with `@`.  OR
+   * Expecting a TSDoc tag starting with `{`.
+   */
+  MissingTag = 'tsdoc-missing-tag',
+
+  /**
+   * A TSDoc tag must be preceded by whitespace.
+   */
+  AtSignInWord = 'tsdoc-at-sign-in-word',
+
+  /**
+   * Expecting a TSDoc tag name after the `@` character (or use a backslash to escape this character).
+   */
+  AtSignWithoutTagName = 'tsdoc-at-sign-without-tag-name',
+
+  /**
+   * Expecting an inline TSDoc tag name immediately after `{@`.  OR
+   * Expecting a TSDoc inline tag name after the `{@` characters.
+   */
+  MalformedInlineTag = 'tsdoc-malformed-inline-tag',
+
+  /**
+   * A TSDoc tag name must start with a letter and contain only letters and numbers.
+   */
+  MalformedTagName = 'tsdoc-malformed-tag-name',
+
+  /**
+   * A TSDoc tag must be followed by whitespace.
+   */
+  TextAfterTag = 'tsdoc-text-after-tag',
+
+  /**
+   * The TSDoc inline tag name is missing its closing `}`.
+   */
+  InlineTagMissingRightBrace = 'tsdoc-inline-tag-missing-right-brace',
+
+  /**
+   * The `{` character must be escaped with a backslash when used inside a TSDoc inline tag.
+   */
+  InlineTagUnescapedBrace = 'tsdoc-inline-tag-unescaped-brace',
+
+  /**
+   * Unexpected character after declaration reference.
+   */
+  InheritDocTagSyntax = 'tsdoc-inheritdoc-tag-syntax',
+
+  /**
+   * The `@link` tag content is missing.
+   */
+  LinkTagEmpty = 'tsdoc-link-tag-empty',
+
+  /**
+   * The ___ character may not be used in the link text without escaping it.
+   */
+  LinkTagUnescapedText = 'tsdoc-link-tag-unescaped-text',
+
+  /**
+   * Unexpected character after link destination.
+   */
+  LinkTagDestinationSyntax = 'tsdoc-link-tag-destination-syntax',
+
+  /**
+   * The URL cannot be empty.  OR
+   * An `@link` URL must begin with a scheme comprised only of letters and numbers followed by `://`.  OR
+   * An `@link` URL must have at least one character after `://`.
+   */
+  LinkTagInvalidUrl = 'tsdoc-link-tag-invalid-url',
+
+  /**
+   * The declaration reference appears to contain a package name or import path, but it is missing the `#` delimiter.
+   */
+  DReferenceMissingHash = 'tsdoc-dreference-missing-hash',
```

Comment:
>dreference [](start = 33, length = 10)

This looks like `dereference`. Maybe change this to just say `reference`?

Reply:
Fixed